server/auth.ts

```
@@ -2,63 +1,63 @@
+      // Clear cookie
+      res.cookie('token', '', { expires: new Date() })
+      return next(createError(401, 'Token expired. Please login again.'))
+    }
+
+    // Other error
+    Logger.error('token error: ')
+    Logger.error(error)
+
+    // Clear cookie
+    res.cookie('token', '', { expires: new Date() })
+    return next(createError(401, 'Invalid token. Please login again.'))
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)